geo-types CI is failing

Relevant logs:

```
   Compiling arbitrary v1.1.6
error[E0658]: use of unstable library feature 'array_from_fn'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/arbitrary-1.1.6/src/lib.rs:655:36
    |
655 |         crate::size_hint::and_all(&array::from_fn::<_, N, _>(|_| {
    |                                    ^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #89379 <https://github.com/rust-lang/rust/issues/89379> for more information

error[E0747]: constant provided when a type was expected
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/arbitrary-1.1.6/src/lib.rs:655:56
    |
655 |         crate::size_hint::and_all(&array::from_fn::<_, N, _>(|_| {
    |                                                        ^
```
 
Any ideas on the reason?

--- a/geo-types/CHANGES.md
+++ b/geo-types/CHANGES.md
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* Pin `arbitrary` version to 1.1.3 until our MSRV catches up with its latest release 
 * Add `point.x_mut()` and `point.y_mut()` methods on `Points`
 
 ## 0.7.7

--- a/geo-types/Cargo.toml
+++ b/geo-types/Cargo.toml
@@ -20,7 +20,7 @@ use-rstar_0_9 = ["rstar_0_9", "approx"]
 
 [dependencies]
 approx = { version = ">= 0.4.0, < 0.6.0", optional = true }
-arbitrary = { version = "1", optional = true }
+arbitrary = { version = "=1.1.3", optional = true }
 num-traits = "0.2"
 rstar_0_8 = { package = "rstar", version = "0.8", optional = true }
 rstar_0_9 = { package = "rstar", version = "0.9", optional = true }

